.cirrus.yml

```diff
@@ -71,7 +71,7 @@ debian_egg_task:
 fedora_selftests_task:
     selftests_script:
        - make develop
-       - PATH=$HOME/.local/bin:$PATH LANG=en_US.UTF-8 AVOCADO_CHECK_LEVEL=0 python3 selftests/check.py --disable-static-checks
+       - PATH=$HOME/.local/bin:$PATH LANG=en_US.UTF-8 AVOCADO_CHECK_LEVEL=0 python3 selftests/check.py
```

Comment:
This is causing failures in the Cirrus CI jobs.  They are run on an environment as close as possible to the RPM build environment, and they lack the lint tools.  Given the logic was inverted from "include" to "exclude", we have to include all but the `--static-check` options here.